net_unix.go

```diff
@@ -81,101 +68,103 @@ type NetUnix struct {
 	Rows []*NetUnixLine
 }
 
-// NewNetUnix returns data read from /proc/net/unix.
-func NewNetUnix() (*NetUnix, error) {
-	fs, err := NewFS(DefaultMountPoint)
-	if err != nil {
-		return nil, err
-	}
-
-	return fs.NewNetUnix()
+// NetUnix returns data read from /proc/net/unix.
+func (fs FS) NetUnix() (*NetUnix, error) {
```

Comment:
Since I'd like to drop the "New" prefix since methods are not constructors, is this also a suitable time to rename this function and types to `NetUNIX`?

Reply:
Yes, I think that's fine since the names are changing anyway.